openlibrary/templates/account/books.html

```diff
@@ -100,6 +100,7 @@ <h3>
                 <li><a class="list-link" href="$(lst.key)">$lst['name'] ($(len(lst.seeds)))</a></li>
             </div>
           </ul>
+        <p class="sansserif large"><a href="/account/import">$_("Import & Export Options")</a></p>
```

Comment:
To be consistent with the other sidebar sections. I think that the import/export section of the sidebar should be an unordered list with a subsection list item.  The subsection could say something like "Import / Export".  It would be good if the new `<ul>` element had a unique class name ending with `-lists`, but used the same CSS rules as the `user-lists` class.

Keeping all of these sections structured similarly will make things simpler when and if we convert this to a Vue component in the future.

Reply:
Sorry for the delay on this one.
I have updated the PR.
![image](https://user-images.githubusercontent.com/64412143/123043282-b15d1000-d415-11eb-94e2-f7b4586d4194.png)